torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -38,6 +38,81 @@
     _linear_args = [Arg() for _ in range(6)]
     _conv_transpose_args = [Arg() for _ in range(11)]
 
+    def _is_valid_group_gemm_fusion(computation_nodes):
+        computation_op = mkldnn._linear_pointwise.default
+        assert all(node.target == computation_op for node in computation_nodes)
+        act = computation_nodes[0].args[0]
+        wgt = computation_nodes[0].args[1]
+        if any(
+            (
+                node.args[0] != act
+                or (node.args[1] == wgt and gemm_idx != 0)
+                or node.args[2]  # <TODO> support bias through epilogue fusion
+                or node.args[1].meta.get("val").dtype != torch.bfloat16  # type: ignore[union-attr]
```

Comment:
Why is it bfloat16 specific?

Reply:
I think it's due to we haven't tested for the other data types (int8 for example) yet. If the general implementation in this PR and the following epilogues fusion looks good. I have tested for other data types and remove this check.